cdap-docs/_common/common-build.sh

```diff
@@ -236,12 +225,6 @@ function make_zip() {
   zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/* --exclude .DS_Store
```

Comment:
Why? Because that's what I was asked to do. People requested a zip file of the documentation.